packages/cli/src/utils/get-json.ts

```diff
@@ -0,0 +1,49 @@
+import fs from "fs";
+import path from "path";
+
+export const TRNSPRNCY_PATH = "@/components/ui/trnsprncy";
```

Comment:
is this always going to be the same path? if not let's change the variable to be something like DEFAULT_TRNSPRNCY_PATH. this way we know that it is the default version.

Reply:
changed it to DEFAULT_TRNSPRNCY_PATH!